### PR TITLE
Install

### PR DESCRIPTION
**Pull request by mgkwill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,50 +43,62 @@ The output should contain something like this `/home/user/lava`
...
 #### [Linux/MacOS]
 ```bash
-$ git clone git@github.com:lava-dl/lava.git
-$ cd lava-dl
-$ pip install -r build-requirements.txt
-$ pip install -r requirements.txt
-$ export PYTHONPATH=$PYTHONPATH:$(pwd)/src
-$ pyb -E unit
+cd $HOME
+git clone git@github.com:lava-dl/lava.git
```
Comment (mgkwill) [inline README.md:47]: This should be `lava-dl.git` not `lava.git` but it looks like this has been present for some time.

Comment (mgkwill) [reply README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -46,50 +43,62 @@ The output should contain something like this `/home/user/lava`
...
+cd lava-dl
+pip install "poetry>=1.1.13"
```
Comment (bamsumit) [inline README.md:49]: `python -m venv .venv` missing?

**Review by bamsumit — APPROVED** [review]
Looks good

File: README.md (hunk 2 above)
Comment (mgkwill) [reply README.md:49]: poetry creates .venv in this case.

**Pull request merged**